README.md

```diff
@@ -129,6 +129,7 @@ OpenDAL is an active open-source project. We are always open to people who want
 - [Vector](https://github.com/vectordotdev/vector): A high-performance observability data pipeline.
 - [OctoBase](https://github.com/toeverything/OctoBase): the open-source database behind [AFFiNE](https://github.com/toeverything/affine), local-first, yet collaborative.
 - [Pants](https://github.com/pantsbuild/pants): A fast, scalable, user-friendly build system for codebases of all sizes.
+- [Questdb](https://github.com/questdb/questdb): An open-source time-series database for high throughput ingestion and fast SQL queries with operational simplicity.
```

Comment:
Please add to the Java Binding section, just like the C Binding.

Reply:
> Please add to the Java Binding section, just like the C Binding.

QuestDB uses both rust core (for replication) and java binding (for test).

---

By the way, please take the branding carefully. It's `QuestDB` instead of `Questdb`.

Reply:
> Please add to the Java Binding section, just like the C Binding.

So also add to Java Binding？